### DISCUSSION TITLE
Wrong Plots Count ?

### DISCUSSION THREAD

**Discussion by eksion** [swar/Swar-Chia-Plot-Manager#601] [Q&A]
Hello

I'm using swar on ubuntu
I'm using 3x1tb ssd ( Raid One ) and try to do the plot

But it seems the plot count is not right.. 
I only have 5 parallel job running but it shows i have 10 plots running

![IMG_20210517_205946](https://user-images.githubusercontent.com/25521335/118501635-24a69e80-b753-11eb-8610-1269a050040e.jpg)

Am i missing something?

_Originally posted by @indowebdeveloper in https://github.com/swar/Swar-Chia-Plot-Manager/discussions/481_

**Comment by swar** [comment]
It's active plots. Not total plots.

Try version 0.1.0 to see if it fixes your issues.
https://github.com/swar/Swar-Chia-Plot-Manager/releases/tag/v.0.1.0
